eacl/target.go

```diff
@@ -77,6 +100,43 @@ func SetTargetECDSAKeys(t *Target, pubs ...*ecdsa.PublicKey) {
 	t.SetBinaryKeys(binKeys)
 }
 
+// SetTargetAccounts sets accounts in Target.
+func SetTargetAccounts(t *Target, accs ...util.Uint160) {
```

Comment:
suggest to make like in this [PR](https://github.com/nspcc-dev/neofs-sdk-go/pull/605) - provide constructors `TargetByAccounts`/`TargetByUsers`, do not add this and `SetTargetUsers`. Test functions may also be reflected

Reply:
These methods use existing `t *Target` to extend it. Constructor is not a good fit for this goal